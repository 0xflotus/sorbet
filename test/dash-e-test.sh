#!/bin/bash
set -eux

diff -u test/dash-e-test.out <(main/ruby-typer -e 'class Foo; end' -p name-table)
