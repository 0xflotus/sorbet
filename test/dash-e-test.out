class <constant:<root>> < ::<constant:Object> () -> <Class:<root>> @ -e:1
  class <constant:Foo> < ::<constant:Object> () -> <Class:Foo> @ -e:1
  class <singleton class:<constant:Foo>> < ::<singleton class:<constant:Object>> () @ -e:1

